packages/form-state-manager/src/utils/manager-api.ts

```diff
@@ -298,7 +298,13 @@ const createManagerApi: CreateManagerApi = ({ onSubmit, clearOnUnmount, initiali
 
   function getFieldState(name: string): AnyObject | undefined {
     if (state.fieldListeners[name]) {
-      return state.fieldListeners[name].state;
+      return {
+        ...state.fieldListeners[name].state,
+        ...state.fieldListeners[name].state.meta,
```

Comment:
I am not sure if FF returns change blur and focus on the getFieldsState method. If it does I believe it should be prefixed by `on`.